### PR TITLE
Stage working

### PR DESCRIPTION
**Pull request by christroutner**
This commit backs out commits 1e09b61 and afaa91d in the `stage` branch because they are breaking the build. This commit also fixes the Swagger GUI for the Wormhole endpoints in the Raw Transactions library, as well as `balancesForAddress` and `balancesForId` from the Data Retrieval library.

### CODE REVIEW

**Comment by christroutner** [comment]
Hmm... I thought I could merge this PR into `stage` and remove those two bad commits, but it doesn't look like GitHub likes that. 

At this point this branch `stage-working` would be a replacement for `stage`. It would be better to delete the `stage` branch and add any code from those two breaking commits into this branch.

File: public/bitcoin-com-rest-v1.json [hunk 1]
```diff
@@ -1551,11 +1469,65 @@
         }
       }
     },
-    "/dataRetrieval/balancesHash/{propertyId}": {
+    "/dataRetrieval/balancexForAddress/{address}": {
```
Comment (cgcardona) [inline public/bitcoin-com-rest-v1.json]: typo?

File: public/bitcoin-com-rest-v1.json [hunk 2]
```diff
@@ -1551,11 +1469,65 @@
...
+          "200": {
+            "description": "successful response"
+          },
+          "400": {
+            "description": "Invalid address. Note: use cashAddress"
+          }
+        }
+      }
+    },
+    "/dataRetrieval/balancesHash/{propertyId}": {
```
Comment (christroutner) [inline public/bitcoin-com-rest-v1.json]: No typo. It's just GitHub's crappy diff.

**Comment by cgcardona** [comment]
:tada: This PR is included in version 1.0.0 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**